modules/calib/src/calibration.cpp

```diff
@@ -655,11 +655,11 @@ static double stereoCalibrateImpl(
     int NINTRINSIC = CALIB_NINTRINSIC;
 
     double dk[2][14]={{0}};
-    Mat Dist[2];
+    std::vector<Mat> Dist(2);
```

Comment:
It makes sense to capture arrays in lambda by reference (at least for performance)

Reply:
So, we don't need `std::vector` changes here.

Ref: https://github.com/alalek/opencv/commit/25fbe9b55df9ae557dc40fc0e7c4d29e3a666f3e